pkg/ssh/exec.go

```diff
@@ -85,6 +87,20 @@ func Exec(ctx context.Context, remotePort int, tty bool, inR io.Reader, outW, er
 		}
 	}
 
+	if agentFwd {
+		sockEnvVar, ok := os.LookupEnv("SSH_AUTH_SOCK")
+		if !ok {
+			log.Info("SSH_AUTH_SOCK is not set, not forwarding socket")
+		} else {
+			if err := agent.ForwardToRemote(connection, sockEnvVar); err != nil {
+				log.Error(err)
```

Comment:
since the execution will keep going, could you make this a `log.Infof` instead? And also add a message for context? Maybe something like `log.Infof("failed to forward ssh agent to remote: %w", err)`. Same with the one below.

Reply:
If I do this, `go vet` complains with:
```
Infof call has error-wrapping directive %w
```
any suggestions?

Reply:
use the `%s` directive instead.